charts/seed-monitoring/charts/prometheus/templates/blackbox-exporter-config.yaml

```diff
@@ -12,3 +12,15 @@ data:
       tcp_vpn:
         prober: tcp
         timeout: 2s
+      http_apiserver:
+        prober: http
+        timeout: 10s
+        http:
+          headers:
+            Accept: "*/*"
+            Accept-Language: "en-US"
+          tls_config:
+            ca_file: "/var/run/secrets/blackbox/ca.crt"
+            cert_file: "/var/run/secrets/blackbox/prometheus.crt"
+            key_file: "/var/run/secrets/blackbox/prometheus.crt"
```

Comment:
The key_file should be ```prometheus.key```

Reply:
Is added and tested
![screenshot 2018-11-14 at 08 37 38](https://user-images.githubusercontent.com/30324854/48467231-98653a80-e7e8-11e8-8d82-321986d8edd6.png)